tests/Extensions/Abstractions.UnitTests/Activities/TimedScopeTests.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using Microsoft.Omex.Extensions.Abstractions.Activities;
+using Microsoft.Omex.Extensions.Abstractions.Activities.Processing;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Microsoft.Omex.Extensions.TimedScopes.UnitTests
+{
+	[TestClass]
+	public class TimedScopeTests
+	{
+		[TestMethod]
+		public void Constructor_WorksProperly()
+		{
+			CreateTimedScope();
+		}
+
+
+		[TestMethod]
+		public void Start_StartsActivity()
+		{
+			TimedScope scope = CreateTimedScope();
+
+			Assert.IsNull(scope.Activity.Id);
+
+			scope.Start();
+
+			Assert.IsNotNull(scope.Activity.Id);
+		}
+
+
+		[TestMethod]
+		public void Stop_MultipleCallsIgnored()
```

Comment:
>Stop_MultipleCallsIgnored [](start = 14, length = 25)

I'd format this method as `Stop_WhenCalledMultipleTimes_IgnoresSuperfluousCalls()` or something similar, to adhere to the tripartite naming scheme we favour. This can also be applied below and to other methods

Reply:
Updated names for both tests, thanks